docs/feature-gates.md

```diff
@@ -254,3 +255,21 @@ More documentation will be coming in the future.
 
 At the moment, Antrea can only create secondary network interfaces using SR-IOV
 VFs on baremetal Linux Nodes.
+
+### ServiceExternalIP
+
+The `ServiceExternalIP` feature enables a controller which can allocate external
+IPs for Services with type `LoadBalancer`. External IPs are allocated from an
+`ExternalIPPool` resource and each IP gets assigned to a Node selected by the
+`nodeSelector` of the pool automatically. That Node will receive Service
+traffic destined to that IP and handle load-balancing (through kube-proxy or
+AntreaProxy depending on the Antrea configuration). To enable external IP
```

Comment:
Unfortunately, there an issue with AntreaProxy proxyAll mode in this release.

Reply:
I wasn't aware of this current limitation. It would be good to have an issue for this.